README.md

```diff
@@ -68,9 +76,35 @@ To compile the contracts:
 ./go contracts:compile
 ```
 
-## Testing
+---
+### Run
+To deploy instances of the contracts for local development without prior knowledge Ganache and Truffle, run the following command:
+```shell
+./go contracts:run
+```
+
+This command starts up Ganache on a random port and migrates all contracts to the Ganache instance. The Ganache instance will remain running the the background, even though the command prompt is available.
+The pid of the Ganache instance is written to a file in the run/pid directory.
+
+To stop the Ganache instance, run the folloing command:
+```shell
+./go ganache:stop
+```
+
+If preferred by those who are familiar with Truffle and Ganache, Ganache can be started up manually in a terminal using the command (or other desired port):
```

Comment:
I've understood "truffle migrate" will only work if the current port is 8545, no?
So in case the user needs to choose another port, he/she would have to change the truffle-config.js file accordingly?

Reply:
Yes, but the command to start Ganache just above specifies port 8545. If someone wants to start Ganache on a different port, they can run the migrate command using the test network or specify their own network in truffle.congi.js Anyone who is familiar with Truffle and Ganache will know this. The README isn't meant to be a course in Truffle and Ganache. That's why the `./go contracts:run` is provided for those who don't know how to use it.

Reply:
My comments was referring to the mention "(or other desired port)" at the end of the sentence, that should confuse people because if they decide to run on another port, the next command "truffle migrate" will not work anymore.
So either you can remove this additional mention, or explain what to be changed if the user wants to set another port for the "truffle migrate" to work

Reply:
Removed (or other desired port)